09_2024_scikit_learn_priorities.md

```diff
@@ -0,0 +1,260 @@
+open-source projects.
+
+### [`scikit-learn`](https://github.com/scikit-learn/scikit-learn)
+
+In a previous [blog
+post](https://papers.probabl.ai/scikit-learns-priorities-at-probabl), we outlined
+Probabl's priorities for `scikit-learn`. In this section, we report on the progress of
+these priorities.
+
+- Short-term priorities
+
+   1. Website Migration: We successfully migrated the `scikit-learn` website to the new
+      PyData theme-based website. This modernization effort improves the user
+      experience and aligns our documentation with the broader PyData ecosystem.
+   1. SLEP6 - Metadata Routing: We completed adapting most estimators to handle
+      metadata routing as specified in
+      [SLEP6](https://scikit-learn-enhancement-proposals.readthedocs.io/en/latest/slep006/proposal.html).
+      This new API consistently handles metadata (e.g., `sample_weight`, `groups`, etc.)
+      for almost all estimators. It allows for fewer bugs and enables new
+   use cases that were not possible before.
+
+Both of these projects represent major milestones for `scikit-learn`. In our current
+round of priorities, we outline follow-up work for both the website and metadata
+routing to further improve and expand on these changes.
+
+-  Mid-term priorities
+
+   1. Implementation of a new `scikit-learn` estimator, called
+      `TunedThresholdClassifierCV`, allowing optimization of the operational decision.
+   1. Progress on the Array-API adoption by making it possible to train and test a
+      pipeline containing a `PCA` and `Ridge` estimator trained on `torch` tensors
+      located on GPU.
+   1. Some work to improve the developer experience by improving the developer tools
+      available in `scikit-learn` and made available for third-party projects.
+
+- Long-term priorities
+
+   1. General maintenance of the project: We reviewed a substantial number of pull
+      requests and issues opened by external contributors.
+   1. Python 3.13 support: We worked on supporting Python 3.13 free-threaded mode and
+      set up regular testing to ensure compatibility until it becomes officially
+      available.
+   1. Releases: Since February 2024, we released `scikit-learn` 1.5.0, 1.5.1, and 1.5.2.
+   1. Continuous Integration and Continuous Deployment: We kept our continuous
+      integration and continuous deployment infrastructure up to date.
+   1. Community outreach: Since February 2024, we attended multiple Python conferences
+      to promote the latest work on `scikit-learn`. We attended the following
+      conferences: PyCon Lithuania, PyCon Italia, CZI Open Science, EuroSciPy, PyData
+      Amsterdam, and PyData Paris.
+
+### Computing orchestration
+
+We dedicated resources to maintain the following projects that are related to computing
+orchestration: `joblib`, `loky`, `cloudpickle`, `threadpoolctl`, and support for
+Python free-threaded mode.
+
+`threadpoolctl` has been released in the past few months and includes better support
```

Comment:
This sounds as if it was released the first time.

Reply:
What do you mean exactly? The support for FlexiBLAS has only been added lately.

Reply:
What you mean to say is "A new version of theadpoolctl has been released".